portfolio/src/main/webapp/script.js

```diff
@@ -18,10 +18,14 @@
 
 let slideIndex = 0;
 
+let sizeStateBig = false;
```

Comment:
I find this variable name confusing and hard to remember. How about `showLargeImages` or something like that?

Reply:
Sounds good, changed in PR 7